src/main/java/org/threadly/concurrent/SingleThreadScheduler.java

```diff
@@ -246,29 +247,40 @@ protected void finalize() {
    * @author jent - Mike Jensen
    * @since 2.0.0
    */
-  protected static class SchedulerManager extends AbstractService 
-                                          implements Runnable {
+  protected static class SchedulerManager implements Runnable {
     protected final NoThreadScheduler scheduler;
+    protected final AtomicInteger state = new AtomicInteger(-1); // -1 = new, 0 = started, 1 = stopping, 2 = stopped
```

Comment:
I hate to waste jar space to define these states as statics....but I know it makes it a little confusing reading through the code.  Do you think I should make these statics or is it okay how it currently is?